dbms/src/main/java/org/polypheny/db/PolyphenyDb.java

```diff
@@ -191,7 +191,7 @@ public void runPolyphenyDb() {
         }
 
         // Configuration shall not be persisted
-        ConfigManager.memoryMode = (mode == PolyMode.TEST || memoryCatalog);
+        ConfigManager.memoryMode = (mode == RunMode.TEST || memoryCatalog);
```

Comment:
Is memory catalog still a thing with the new catalog?

Reply:
Now it is again.